tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -29,6 +31,8 @@ node_modules/chrome-devtools-frontend/front_end/Runtime.js(270,9): error TS2322:
   Type 'void' is not assignable to type 'undefined'.
 node_modules/chrome-devtools-frontend/front_end/Runtime.js(280,5): error TS2322: Type 'Promise<void>' is not assignable to type 'Promise<undefined>'.
 node_modules/chrome-devtools-frontend/front_end/Runtime.js(283,7): error TS2554: Expected 2-3 arguments, but got 1.
+node_modules/chrome-devtools-frontend/front_end/Runtime.js(527,9): error TS2322: Type 'Function' is not assignable to type 'new () => any'.
+  Type 'Function' provides no match for the signature 'new (): any'.
```

Comment:
Function isn't assignable to `new(): any` anymore. Not sure why yet, but *seems* correct?

Reply:
I think something else was hiding this error, because it's in an error in TS and JS, and has been for a while.